controllers/helmchartinstall_controller.go

```diff
@@ -77,61 +126,255 @@ func (r *HelmChartInstallReconciler) render(ctx context.Context, claim *helmv1al
 	- Watch for new types being provided by stacks
 	- When there is a new type provided, watch for instances being created
 	- When an instance is created, feed it into the renderer
+
+	TODO This could use a bit of refactoring so it flows more nicely
 	*/
 
 	// configuration is a typed object
-	configuration, err := r.getStackConfiguration()
+	configuration, err := r.getStackConfiguration(ctx, claim)
+	// TODO check for errors
+
+	targetResourceBehavior, err := r.getBehavior(ctx, claim, configuration)
```

Comment:
https://talks.golang.org/2014/names.slide#6

Go has a novel approach to variable names, in that it prefers shorter variables in many cases. In this instance I know that `targetResourceBehaviour` is probably a behaviour, because it's returned by a method named `getBehaviour`, and only lives for a few lines after it is declared. If this variable were named `tb` I'd probably still understand what it was doing just as well.

Reply:
Cool; seems reasonable to me